dataflow/src/main/java/org/checkerframework/dataflow/reachingdefinitions/ReachingDefinitionsStore.java

```diff
@@ -0,0 +1,116 @@
+package org.checkerframework.dataflow.reachingdefinitions;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.Store;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizer;
+import org.checkerframework.dataflow.expression.JavaExpression;
+import org.checkerframework.javacutil.BugInCF;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.StringJoiner;
+
+/** A reaching definitions store contains a set of reaching definitions represented by nodes. */
```

Comment:
The class LiveVarStore shows `/** A live variable store contains a set of live variables represented by nodes. */`, here I think "node" means the type of the field "liveVariable" of LiveVarValue is Node.
However, for reachingdefintionsValue class, the type is AssignmentNode. So I will change it to AssignmentNode rather than Node if you also agree.

Reply:
`/** A reaching definitions store records information about ReachingDefinitionsValue */`